packages/discord-snippet/0.1.0/package.yml

```diff
@@ -0,0 +1,110 @@
+# espanso configuration file
+
+# This is the default configuration file, change it as you like it
+# You can refer to the official documentation:
+# https://espanso.org/docs/
+
+# Matches are the substitution rules, when you type the "trigger" string
+# it gets replaced by the "replace" string.
+name: discord-snippet
+parent: default
+
+matches:
+  ### ----------------
+  ### Text Formatting
+  ### ----------------
+
+  # Discord Italic
+- trigger: ":dita"
```

Comment:
Whilst your code works, we tend to adopt the YAML convention of indenting block sequences with respect to their headings, e.g.:
```yml
matches:
  - trigger: ":dita"
    replace: "_$|$_"
```
etc..  Would you mind doing the same, please?

Reply:
Thank you.